code/game/machinery/jukebox.dm

```diff
@@ -275,3 +275,38 @@
 	icon_state = state_base
 	if(playing)
 		AddOverlays("[state_base]-running")
+
+/obj/machinery/media/jukebox/gramophone/MouseDrop(atom/over, src_location, over_location, src_control, over_control, params)
+	. = ..()
+	if(use_check(usr) || !Adjacent(usr))
+		return
+	if(!ishuman(usr) && (!isrobot(usr) || isDrone(usr))) //Humans and borgs can collapse, but not drones
```

Comment:
This would make other non-human things (like the gremorian queen) able to collapse it